packages/create-plugin/src/bin/run.ts

```diff
@@ -2,6 +2,10 @@
 
 import minimist from 'minimist';
 import { generate, update, migrate, version } from '../commands';
+import { operatingSystemCheck } from '../utils/utils.os';
+
+// Exit early if operating system isn't supported.
+operatingSystemCheck();
```

Comment:
nit: Because this is a disrupting code in the main function I'd change it to


```
if (isNotSupportedPlatform()){
   exit(1);
}
```

So you don't hide away code that can exit the cli in another file where is not clear.